docs/sql/joins.md

```diff
@@ -26,3 +46,376 @@ Open Distro for Elasticsearch SQL supports inner joins, left outer joins, and cr
 
 1. You can't use GROUP BY or ORDER BY for results.
 1. LIMIT with OFFSET (e.g. `LIMIT 25 OFFSET 25`) is not supported.
+
+### Description
+
+The `JOIN` clause combines columns from one or more indices using values common to each.
+
+### Example 1: Inner join
+
+Inner join creates a new result set by combining columns of two indices based on your join predicates. It iterates the two indices and compares each document to find the ones that satisfy the join predicates. You can optionally precede the `JOIN` clause with an `INNER` keyword.
+
+The join predicate(s) is specified by an ON clause.
```

Comment:
Join predicates are specified by the ON clause.

Reply:
Done.